modules/service/src/main/scala/lucuma/odb/graphql/mapping/QueryMapping.scala

```diff
@@ -413,6 +415,36 @@ trait QueryMapping[F[_]] extends Predicates[F] {
       }
   }
 
+  private lazy val ObservingModeGroup: PartialFunction[(TypeRef, String, List[Binding]), Elab[Unit]] =
+    val WhereObservationBinding = WhereObservation.binding(ObservingModeGroupType / "observations" / "matches")
+    {
+      case (QueryType, "observingModeGroup", List(
+        ProgramIdBinding.Option("programId", rPid),
+        ProposalReferenceBinding.Option("proposalReference", rProp),
+        ProgramReferenceBinding.Option("programReference", rProg),
+        WhereObservationBinding.Option("WHERE", rWHERE),
+        NonNegIntBinding.Option("LIMIT", rLIMIT),
+        BooleanBinding("includeDeleted", rIncludeDeleted)
+      )) =>
+        Elab.transformChild: child =>
+          (programPredicate(rPid, rProp, rProg, Predicates.observingModeGroup.program), rWHERE, rLIMIT, rIncludeDeleted).parTupled.flatMap: (program, WHERE, LIMIT, includeDeleted) =>
+            val limit = LIMIT.foldLeft(ResultMapping.MaxLimit)(_ min _.value)
+            ResultMapping.selectResult(child, limit): q =>
+              FilterOrderByOffsetLimit(
+                pred = and(List(
+                  WHERE.getOrElse(True),
+                  program,
```

Comment:
This predicate appears to be ignored.  If I add `program { id }` to the observing mode group query I can be told 

```
java.lang.Throwable: Expected single value for field 'id' of type Program at List(program, matches, observingModeGroup), found many
```

but it is unclear why.